ES(Spanish)/README.md

```diff
@@ -1,15 +1,18 @@
-# :wave:  Bienvenidos al Código para la Comunidad
+# :wave:  Code For Community te da la bienvenida
+
 <p align="center"><img src="https://user-images.githubusercontent.com/75534912/194343020-6cbd2485-c2ea-4779-87e2-9708238398e7.png"></p>
 
-# La entrada del código abierto | Los mejores recursos de la programación
-## Nuestros objetivos:rocket:
-Los objetivos de la comunidad no son solo proveer los mejores recursos, sino también <b>motivatar y animar los principantes que no han contruibido nunca al código abierto, y ayudarlos a empezar</b>proyectos fácilmente y con apoyo total.
+# La entrada al código abierto | Los mejores recursos de programación
+
+## Nuestros objetivos:rocket
+
```

Comment:
In line 7 :point_down: 
```
## Nuestros objectivos :rocket:
```

Reply:
I remove the empty line spaces, sorry for that. They were made for the markdownlint extension for the VSCode. 
Also, I made a last fix in the last line.